src/rules/prefer-invoke.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @fileoverview Rule to check if a call to map should be a call to invokeMap
+ */
+"use strict"
+const _ = require("lodash")
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {}
+  },
+
+  create(context) {
+    const { getLodashMethodVisitors } = require("../util/lodashUtil")
+
+
+    const getParentIfStatement = n => {
+      if (_.get(n, 'parent.type') === "IfStatement") {
```

Comment:
You're not checking that the current node is the `test` part and not the `subsequent` or `alternate`.
For instance, `getParentIfStatement` returns a node for
```js
if (something)
  _.isFunction(x)
```

Reply:
@ganimomer This is the last note so I just wanna clarify before fixing this. What we want is to return a node only if  the `test` part of the If statement contains an `isFunction` call. Did I get this right?

Reply:
Yes. And if the condition is a conjunction (`&&`), the `_.isFunction` should be the last condition, like we said.